scripts/pi-hole/php/footer.php

```diff
@@ -49,15 +49,15 @@
     <footer class="main-footer">
 	<!-- Version Infos -->
         <div class="pull-right hidden-xs hidden-sm<?php if(isset($core_commit) || isset($web_commit) || isset($FTL_commit)) { ?> hidden-md<?php } ?>">
-            <b>Pi-hole Version </b> <?php
+            <b>Pi-hole Version</b> <?php
             echo $core_current;
             if(isset($core_commit)) { echo " (".$core_branch.", ".$core_commit.")"; }
-            if($core_update){ ?> <a class="alert-link lookatme" href="https://github.com/pi-hole/pi-hole/releases" target="_blank">(Update available!)</a><?php } ?>
-            <b>Web Interface Version </b><?php
+            if($core_update){ ?> <a class="alert-link lookatme" href="https://github.com/pi-hole/pi-hole/releases" target="_blank">(Update available!)</a><?php } ?> &bullet;
+            <b>Web UI Version</b> <?php
```

Comment:
You can remove the word `UI` even.

Reply:
Ah, OK. I'd used "Web UI" because I also called it that one of the Settings tabs, because it fit better.